.github/workflows/tag-version.yml

```diff
@@ -0,0 +1,102 @@
+          under a bot user account only used for such operations, so it's not tied to a natural person.
+    inputs:
+      path-include-list:
+        description: >
+          PowerShell string array of paths, relative to the repository root, to search for GHA files, e.g.
+          '@(".github")' or '@(".github/actions", ".github/workflows")'. The parameter must be a PowerShell string
+          array.
+        required: false
+        default: '@(".github")'
+        type: string
+      file-include-list:
+        required: false
+        default: '@("*.yml","*.yaml")'
+        description: >
+          PowerShell string array of file name patterns to include when evaluating GHA files, e.g. '@("*.yml")' or
+          '@("*.yml", "*.yaml")'. The parameter must be a PowerShell string array.
+        type: string
+      called-repo-base-include-list:
+        required: false
+        default: '@("${{ github.repository }}")'
+        description: >
+          PowerShell string array of repository base URLs to include when evaluating called GHA Workflows and Actions,
+          e.g '@("Lombiq/GitHub-Actions")' or '@("Lombiq/GitHub-Actions",
+          "Lombiq/Open-Source-Orchard-Core-Extensions")'. The parameter must be a PowerShell string array.
+        type: string
+      additional-pattern-include-list:
+        required: false
+        default: '@()'
+        description: >
+          PowerShell string array of additional RegEx patterns to include when searching for branch references that need
+          to be updated, e.g.
+          'https://raw.githubusercontent.com/Lombiq/GitHub-Actions/(?<ref>[aA-zZ,0-9,-,\.,/,_]*)/.github/'. The pattern
+          MUST include a regex named capture group (?<ref>[aA-zZ,0-9,-,\.,/,_]*) so the captured ref can be updated
+          correctly. The parameter must be a PowerShell string array.
+        type: string
+      expected-ref:
+        required: false
+        default: '${{ github.ref_name }}'
+        description: The expected reference value to set for all called GHA Workflows and Actions.
+        type: string
+
+jobs:
+  run:
+    name: Set GitHub Action/Workflow to Version Tag
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: pwsh
+    steps:
+      - name: Setup Scripts
+        run: |
+          "${{ github.action_path }}" >> $Env:GITHUB_PATH
+          (Resolve-Path "${{ github.action_path }}/../../../Scripts").Path >> $Env:GITHUB_PATH
+
+      - name: Checkout Repository
+        uses: Lombiq/GitHub-Actions/.github/actions/checkout@dev
+        with:
+          token: ${{ secrets.TAG_VERSION_TOKEN }}
+
+      - name: Determine Version Tag Name from Branch Name
+        id: determine-tag
+        run: |
+          $refname = "${{ inputs.expected-ref }}"
+          $tagname = $refname -replace 'release/', ''
+          Set-GitHubOutput -Key 'output' -Value $tagname
```

Comment:
```suggestion
          Set-GitHubOutput -Key 'tagname' -Value $tagname
```

Reply:
Fixed in 36bbcefab51d7f5a744a026b06f1a30f685f6bf5.